test/unit/events/replayemitter.spec.js

```diff
@@ -2,108 +2,108 @@ import { expect, spy, use } from 'chai'
 import chaiSpies from 'chai-spies'
 import ReplayEmitter from '../../../src/events/replayemitter'
 
-use(chaiSpies);
+use(chaiSpies)
```

Comment:
should we update the `package.json` to run lint on the tests? The Builder is already doing the test linting https://github.com/LiveIntent/LiveConnectBuilder/pull/1/commits/101b416b54bd86d296a7588db57e1d688096966d

Reply:
Yes, and no... however if i run lining on specs, then i'm getting a lot of complaints about describes and its.
I think that's the reason why it's not enabled in the first place